mbed_tools_ci_scripts/create_news_file.py

```diff
@@ -55,6 +55,8 @@ def determine_news_file_path(news_type: NewsType) -> pathlib.Path:
 
 def _write_file(file_path: pathlib.Path, text: str) -> None:
     file_path.parent.mkdir(parents=True, exist_ok=True)
+    if not text.endswith("\n"):
```

Comment:
Why `if`? Can't we always add the `\n`? Don't the tests in the next commit always assume a `\n` is added?

Reply:
If we add two newlines then `splitlines` returns a list of `len() == 2` and the `assert-news` check will fail.

Reply:
https://github.com/ARMmbed/mbed-tools-ci-scripts/blob/master/mbed_tools_ci_scripts/assert_news.py#L47

Reply:
But we probably should test it doesn't add a newline if one already exists. I'll add another test case for that.

Reply:
`If we add two newlines` but we wouldn't add two newlines. Is there a case where we would?

Reply:
If you give "my newsfile message\n" to it on the command line :)

Reply:
bah, who does that :p